MekHQ/src/mekhq/campaign/unit/Unit.java

```diff
@@ -3741,10 +3779,42 @@ private void calcCompositeCrew() {
             if(p.getHits() > 0 && !usesSoloPilot()) {
                 continue;
             }
-            if(p.hasSkill(gunType)) {
-                sumGunnery += p.getSkill(gunType).getFinalSkillValue();
-                nGunners++;
+
+            if (SkillType.isRPGGunnery(gunType) &&
+                    campaign.getGameOptions().booleanOption(OptionsConstants.RPG_RPG_GUNNERY)) {
```

Comment:
These should be in the opposite order, always check if you are using the RPG gunnery option first.